### PR TITLE
Feat/add ruby snippets

### PR DESCRIPTION
**Pull request by eugenioLopezRamos**
Add ruby snippets for the SDKs done to date

### CODE REVIEW

File: referencia/webpayrest/README.md [hunk 1]
```diff
@@ -1685,7 +1786,7 @@ response_url  <br> <i> String </i> | URL del comercio a la cual Webpay redirecci
 ```
 
 ```ruby
-# Este SDK aún no se encuentra disponible
```
Comment (afiebig) [inline referencia/webpayrest/README.md]: What is the reason of this change?

Comment (afiebig) [reply referencia/webpayrest/README.md]: Is this not available yet?

File: referencia/webpayrest/README.md [hunk 2]
```diff
@@ -1737,7 +1838,7 @@ realizar las transacciones de pago.
...
+  # Este SDK aún no se encuentra disponible
```
Comment (afiebig) [inline referencia/webpayrest/README.md]: ?

File: referencia/webpayrest/README.md [hunk 3]
```diff
@@ -1827,7 +1928,7 @@ Permite eliminar un usuario enrolado a Oneclick.
 ```
 
 ```ruby
-# Este SDK aún no se encuentra disponible
+#  Este SDK aún no se encuentra disponible
```
Comment (afiebig) [inline referencia/webpayrest/README.md]: ?

**Comment by a1b4** [comment]
@eugenioLopezRamos can you change thsi PR to Draft PR because we can merge it yet.
